### PR TITLE
Increase slider tails' worth to 150 points (up from 30)

### PR DESCRIPTION
**Pull request by smoogipoo**
@Bubbleman532, @Zyfarok, and others in the community have mentioned that the changes to sliders (tails not affecting the slider's main judgement, because that judgement is now on the head) have made tails not worth as much anymore, and removes the need to aim sliders on higher SR maps that very rarely have any ticks in sliders.

This PR makes tails worth 150 points, compared to the 30 points of a normal tick.

Not sure on the exact value this should be, was given a range anywhere between 100 and 230.

### CODE REVIEW

**Comment by smoogipoo** [comment]
Pinging @bdach in case you have knowledge on anything that reduces the chance of this passing review. You previously brought up https://github.com/ppy/osu/pull/25732#discussion_r1423476903, but at the very least this shouldn't have the same issue...

**Comment by Zyfarok** [comment]
Tl;dr of my answer to peppy: yes 150 is good and safe.

This PR can probably close https://github.com/ppy/osu/issues/23403

(I was originally considering also slightly boosting slider-ticks/repeats but bubbleman and others convinced me that this would cause issues with buzz-sliders and other high-tick-rate sliders. The combo-break is already enough to make them worth aiming)

File: osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public class LegacyTailJudgement : OsuJudgement
 
         public class TailJudgement : SliderEndJudgement
         {
-            public override HitResult MaxResult => HitResult.LargeTickHit;
+            public override HitResult MaxResult => HitResult.SliderTailHit;
```
Comment (bdach) [inline osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs:32]: Just to confirm, this is 100% intended to only affect non-classic mod scores?

**Comment by bdach** [comment]
As far as I can tell, we can apply this immediately without any issues, as long as we're starting with a clean slate of lazer scores - which I _understand_ we are (let me know if I'm wrong). We aren't doing anything to even attempt estimation of impact of slider ends in the current total score conversion code for legacy scores, and slider ends themselves do not affect accuracy for legacy scores. So from that angle this _appears_ to be fine.

This is also introducing a brand new hit result type which _should_ not lead to any issues on the infra side as much as it looks ugly, but I think there's still a possibility to fix this ugliness later if we ever do that refactor which would let rulesets basically define their own hit types.

That said if we ever decide to back down from this, then that's when the issues will start. The easiest way to address those would be a score wipe, but I'm not sure if that'll be palatable. But maybe that won't be required...

**Comment by smoogipoo** [comment]
We will already be wiping scores, basically. At least they won't be present on the leaderboard/give pp.

File: osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs (hunk 1 above)
Comment (peppy) [reply osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs:32]: Yeah, this would be the intention. `LargeTickHit` matches classic scoring closer.

Comment (smoogipoo) [reply osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs:32]: I think @peppy is misunderstanding. Classic mod is using `SmallTickHit`, and the reason it is so is because the slider judgment (the proportional one) is the one that applies the combo. Classic mod is still not quite 1:1.

So yes, it is intended that this only affects non-classic-mod scores.

**Review by peppy — APPROVED** [review]
